homeassistant/components/climate/sensibo.py

```diff
@@ -75,6 +78,13 @@ def __init__(self, client, data):
         self._id = data['id']
         self._do_update(data)
 
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return (SUPPORT_TARGET_TEMPERATURE | SUPPORT_OPERATION_MODE |
```

Comment:
Sensibo's supported features depend on the A/C controlled. Lets proceed as-is and later I'll change the code to properly auto-detect features.

Reply:
Ah, I remember that looking weird. But the logic whether something is supported or not hides in the client, right?